src/products/components/ProductList/ProductList.tsx

```diff
@@ -376,7 +375,10 @@ export const ProductList: React.FC<ProductListProps> = props => {
                       {(!product && <Skeleton />) ||
                         (!product?.channelListings?.length && "-") ||
                         (product?.channelListings !== undefined && channel ? (
-                          <ProductAvailabilityStatusLabel channel={channel} />
+                          <AvailabilityStatusLabel
+                            channel={channel}
+                            type="product"
```

Comment:
`type` has to be localized as well. 
```suggestion
                            type={intl.formatMessage({ defaultMessage: "product" })}
```